bridges/snowbridge/pallets/inbound-queue-v2/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use snowbridge_core::inbound::Proof;
+use snowbridge_router_primitives::inbound::v2::Message;
+use xcm::latest::Xcm;
+
+sp_api::decl_runtime_apis! {
+	pub trait InboundQueueApiV2
+	{
+		/// Dry runs the provided message on AH to provide the XCM payload and execution cost.
+		fn dry_run(message: Message, proof: Proof) -> (Xcm<()>, u128);
```

Comment:
The proof is not necessary for dry-running.

Also the local fee should be the `Balance` template parameter

```suggestion
		fn dry_run(message: Message) -> (Xcm<()>, Balance);
```

Reply:
Addressed in https://github.com/claravanstaden/polkadot-sdk/pull/4/commits/42d688a28a31e76e58043955d3fbef2c5cb4a677.